packages/server-wallet/src/config/defaults.ts

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable no-process-env */
+import {
+  LoggingConfiguration,
+  NetworkConfiguration,
+  OptionalDatabaseConfiguration,
+  OptionalServerWalletConfig,
+  ServerWalletConfig,
+} from './types';
+
+export const defaultDatabaseConfiguration: OptionalDatabaseConfiguration & {
+  connection: {host: string; port: number; user: string};
+} = {
+  pool: undefined,
+  debug: false,
+  connection: {host: 'localhost', port: 5432, user: 'postgres'},
+};
+
+/**
+ * Network config
+ */
+
+export const defaultNetworkConfiguration: NetworkConfiguration = {
+  chainNetworkID: '0x00',
+  erc20Address: undefined,
+  ethAssetHolderAddress: undefined,
+  erc20AssetHolderAddress: undefined,
+};
+
+/**
+ * Logging and Metrics config
+ */
+export const defaultLoggingConfiguration: LoggingConfiguration = {
+  logLevel: 'info',
+  logDestination: 'console',
+};
+
+export const defaultMetricConfiguration = {timingMetrics: false};
+
+/**
+ * Server Wallet config
+ */
+
+/**
+ * These are the default values that will be used by the server wallet
+ * if not overidden in the provided config
+ */
+export const defaultConfig: OptionalServerWalletConfig = {
+  databaseConfiguration: defaultDatabaseConfiguration,
+  networkConfiguration: defaultNetworkConfiguration,
+  loggingConfiguration: defaultLoggingConfiguration,
+  metricConfiguration: defaultMetricConfiguration,
+  // TODO: List addresses this corresponds to
+  ethereumPrivateKey: '0x7ab741b57e8d94dd7e1a29055646bafde7010f38a900f55bbd7647880faa6ee8',
+  skipEvmValidation: false,
+  workerThreadAmount: 10,
+};
+
+export const DEFAULT_DB_NAME = 'server_wallet_test';
+export const DEFAULT_DB_USER = 'postgres';
```

Comment:
I don't think this should be the default either. It seems like the default configuration in _our setup_ (the postgres app on macOS) is leaking into the source code of the wallet.